pkg/otlp/metrics/metrics_translator_test.go

```diff
@@ -673,6 +673,29 @@ func TestMapRuntimeMetricsNoMapping(t *testing.T) {
 	assert.Empty(t, rmt.Languages)
 }
 
+func TestMapSystemMetrics(t *testing.T) {
+	ctx := context.Background()
+	tr, err := NewTranslator(
+		zap.NewNop(),
+		WithRemapping(),
+	)
+	require.NoError(t, err)
+	consumer := &mockFullConsumer{}
+	rmt, err := tr.MapMetrics(ctx, createTestMetricWithAttributes("system.filesystem.utilization", pmetric.MetricTypeGauge, nil, 1), consumer)
+	if err != nil {
+		t.Fatal(err)
+	}
+	startTs := int(getProcessStartTime()) + 1
+	assert.ElementsMatch(t,
```

Comment:
I assume this test was failing before your change.

Reply:
Correct, without the changes in metrics_translator the output is otel.system.filesystem.utilization and **otel**.system.disk.in_use